### ISSUE TITLE
Reduce dependencies (e.g. gulp, phantomjs)

### ISSUE THREAD

**Issue by sglanzer** [johnjmartin/ember-cli-backstop#3]
ember-cli doesn't require phantomjs, but asks the user to install it separately.  We can likely make the assumption that phantomjs is installed, but if not we can include documentation on installing it.  The rest of the dependencies should be evaluated to see if they should receive the same treatment.

**Comment by sglanzer** [comment]
This doesn't seem necessary currently since gulp, phantomjs and casperjs are all required by backstopjs and we're already documenting the global install of phantomjs and casperjs (gulp is included as a dependency and referenced locally - we may want to follow the lead of the backstopjs project and include it as a documented global install as well).

Let's scrub the rest of the dependencies, including those from the default ember-cli addon generation and make a decision on the approach to gulp.  I think we can close this after that.

**Comment by johnjmartin** [comment]
Alright, I've run several times without a global gulp and it doesn't seem to make a difference. Seems like an unnecessary install. What do you think?

**Comment by sglanzer** [comment]
Yeah, it's because we're including it as a dependency and executing against the local copy.  We could avoid that since gulp is a common install and just rely on the global instead.
